src/main/java/uk/gov/laa/ccms/caab/controller/application/section/EvidenceSectionController.java

```diff
@@ -128,7 +128,7 @@ public String viewEvidenceUpload(
 
     // Initialise the form data object.
     final EvidenceUploadFormData evidenceUploadFormData = new EvidenceUploadFormData();
-    evidenceUploadFormData.setApplicationOrOutcomeId(activeCase.getApplicationId());
+    evidenceUploadFormData.setApplicationOrOutcomeId(activeCase.getApplicationId().toString());
```

Comment:
Could do `String.valueOf(activeCase.getApplicationId())` to make this line null safe

Reply:
Updated